spec/lib/rbac/roles_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe RBAC::Roles do
+  let(:api_instance) { instance_double(RBACApiClient::RoleApi) }
+  let(:catalog_admin) { {:name => "Catalog Administrator", :description => "A catalog administrator roles grants read, write and order permissions"} }
```

Comment:
@lindgrenj6 do you want to use an instance_double here
similar to 
https://github.com/ManageIQ/catalog-api/blob/7ae8422696a4cdec369c9e4746a7683f61d4ec6f/spec/support/rbac_shared_contexts.rb#L11

Reply:
I changed it up to use webmocks since that way the only thing we're truly mocking out is the api call. Have a look at it and let me know what you think!